### DISCUSSION TITLE
Nan appears during sd3 training

### DISCUSSION THREAD

**Discussion by jyy-1998** [huggingface/diffusers#8598] [Q&A]
I try to use the  train_dreambooth_sd3.py for training. The loss and results were normal at first, but after two or three thousand steps, Nan appeared. Has anyone had the same problem?

**Comment by asomoza** [comment]
Did you try after or before this [PR](https://github.com/huggingface/diffusers/pull/8528) was merged? If not I suggest you test it with that one. I haven tested for more than 2000 steps but it worked fine for me.
